Website/Portals/_default/HotcakesViews/_default/Views/Cart/MiniCart.cshtml

```diff
@@ -1,74 +1,18 @@
-﻿@model Hotcakes.Modules.Core.Models.CartViewModel
-<script type="text/javascript">
-    $(function () {
-        var $target = $('#hcMiniCartTooltip');
-        var $minicart = $('#hcMiniCart');
-        var $iconbox = $minicart.find(".hc-iconbox");
-
-        if ($target.length) {
-            $minicart.hover(function () {
-                $iconbox.css("background-color", "#FFF");
-                $target.slideDown().position(
-                    'right'
-                );
-            }, function () {
-                $target.slideUp(function () {
-                    $iconbox.css("background-color", "");
-                });
-            });
-        }
-    });
-</script>
-
+﻿@model Hotcakes.Modules.Core.Models.MiniCartViewModel
 <div id="hcMiniCart" class="hc-minicart">
     <div class="hc-iconbox">
         <a href="@Url.RouteHccUrl(HccRoute.Cart)">
-            @Model.CurrentOrder.TotalQuantity
+            @Model.TotalQuantity
         </a>
     </div>
-    @if (!Model.CartEmpty)
+    @if (Model.TotalQuantity > 0)
```

Comment:
This doesn't appear to be a good update for stores that allow zero dollar orders.